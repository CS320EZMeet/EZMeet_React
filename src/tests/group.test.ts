import renderer from 'react-test-renderer';
import Group from '../pages/group';

it('renders group correctly', () => {
    const tree = renderer
        .create( Group() )
        .toJSON();
    expect(tree).toMatchSnapshot();
});