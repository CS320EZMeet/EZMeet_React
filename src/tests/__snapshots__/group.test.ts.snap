// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="groupCard card"
>
  <div
    className="row"
  >
    <div
      className="my-auto mx-auto col-lg-5 col-md-5"
    >
      <img
        className="card-img-top"
        src="Togepi.jpeg"
      />
    </div>
    <div
      className="my-auto mx-auto text-center col"
    >
      <p
        className="card-text"
      >
        Bob
      </p>
    </div>
  </div>
</div>
`;
